.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
+
+(Is your feature request related to a problem?)  
+*Ex: I get frustrated when having to use nested `if` conditions where a `elif` could be used*
```

Comment:
No frustration here.

I would like this have a form of a user story. 

```suggestion
*Ex: As a developer, when using many if statements, I would like to be able to chain them using elif. The current syntax doesn't support this in a readable way.
```

Reply:
Changed it in https://github.com/inmanta/inmanta/pull/2416/commits/2e2f1f6a155d159d30daa2953c86835f4bde579f